account_invoice_merge_payment/README.rst

```diff
@@ -10,7 +12,28 @@ module to support fields added in Account Payment Partner from OCA/bank-payment
 Installation
 ============
 
-* The module is automatically installed when account_invoice_merge and account_payment_partner are installed
+* The module is automatically installed when account_invoice_merge and
+account_payment_partner are installed
+
+Usage
+=====
+
+Select several invoices from the same partner and click on **Merge Partner
+Invoice** in Action button menu.
```

Comment:
Add that the payment mode will be considered also as merging criteria (with different payment modes they won't be merged) and the merged invoice will have the payment mode filled if they are the same.